src/v/utils/delta_for.h

```diff
@@ -31,30 +31,33 @@ static constexpr uint32_t FOR_buffer_depth = 16;
  */
 struct delta_xor {
```

Comment:
>  with a nttp value

what is `nttp`?

Reply:
Acronym for Non-Type Template Parameter. It's a really bad name for a template parameter that is an value and not a type

Reply:
ahh, of course. thanks